intern/service/boss/baseversion.py

```diff
@@ -181,8 +190,12 @@ def build_cutout_url(
         if len(id_list) > 0:
             urlWithParams += '?filter=' + self.convert_int_list_to_comma_sep_str(id_list)
 
-        if no_cache:
-            urlWithParams += '?no-cache=true'
+        if access_mode == 'no_cache':
```

Comment:
Like Derek said above, you shouldn't be comparing to hard-coded strings. That defeats the purpose of using an Enum. We use Enums so if we need to make a change, we only need to change in one place.